sound/snapcast/Makefile

```diff
@@ -0,0 +1,127 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=snapcast
+BASE_VERSION:=0.28.0
+PKG_SOURCE_DATE:=2024-04-15
+PKG_SOURCE_VERSION:=5fa01b8a49d842ed368138ee071cdf1c34be7a64
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/badaix/snapcast.git
+PKG_MIRROR_HASH=bac3da20899f507e096df96892a4292ecad907fbeb158ddd056a5893ae96a04c
+
+PKG_VERSION:=$(BASE_VERSION).$(subst -,.,$(PKG_SOURCE_DATE))~$(call version_abbrev,$(PKG_SOURCE_VERSION))
+
+PKG_MAINTAINER:=
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += -DBUILD_TESTS=OFF
+
+define Package/snapcast/Default
+	SECTION := sound
+	CATEGORY := Sound
+	TITLE := Synchronous multiroom audio player
+	DEPENDS := +AUDIO_SUPPORT:alsa-lib +libstdcpp +libavahi-client +libatomic +libogg +libflac +libopus +boost +libsoxr
```

Comment:
libstdcpp is included with boost. Minor detail.

Reply:
> libstdcpp is included with boost. Minor detail.

Thank you very much! I'm a package maintainer newbie.